lib/ui/window.dart

```diff
@@ -51,18 +51,22 @@ part of dart.ui;
 /// the [viewPadding] anyway, so there is no need to account for
 /// that in the [padding], which is always safe to use for such
 /// calculations.
-///
-/// See also:
-///
-///  * [FlutterWindow], a special case of a [FlutterView] that is represented on
-///    the platform as a separate window which can host other [FlutterView]s.
-abstract class FlutterView {
+class FlutterView {
+  FlutterView._(this._viewId, this.platformDispatcher);
+
+  /// The opaque ID for this view.
+  final Object _viewId;
```

Comment:
Is this intentionally made an `Object` instead of a number?

Reply:
The `viewId` on `FlutterWindow` was an `Object`. I think that using types like string or a number makes sense as a `viewId` so I just left it.